test/functional/testlib.bash

```diff
@@ -1780,22 +1778,47 @@ destroy_test_fs() { # swupd_function
 
 create_config_file() { # swupd_function
 
+	SWUPD_CONFIG_DIR="$("$SWUPD" --version | grep "config file path" | awk '{print $4}')"
```

Comment:
This isn't going to work for paths separated with ':'. like the default path.

Reply:
And I'm a bit concerned on overwriting any possible config file that the user may have.

Reply:
yes, that is right, this won't work if the user has more than one path defined.
And about the possibility of overwriting an config file a user may have,  that should not happen , that is why I'm backing it up first if config file already exists.